nixos/modules/services/misc/nitter.nix

```diff
@@ -155,6 +160,30 @@ in
           description = lib.mdDoc "Use base64 encoding for proxied media URLs.";
         };
 
+        enableRSS = mkOption {
```

Comment:
This should use mkEnableOption

Reply:
Should I also change the following similar entries?